### ISSUE TITLE
Ballerina testStreamingStatePersistence testcase fails intermittently

### ISSUE THREAD

**Issue by Manuri** [ballerina-platform/ballerina-lang#14794]
**Description:**
```bash
[ERROR] testStreamingStatePersistence(org.ballerinalang.stdlib.streams.BallerinaStreamsV2StatePersistenceTest)  Time elapsed: 12.032 s  <<< FAILURE!
java.lang.AssertionError: expected [60] but found [30]
	at org.ballerinalang.stdlib.streams.BallerinaStreamsV2StatePersistenceTest.testStreamingStatePersistence(BallerinaStreamsV2StatePersistenceTest.java:89)

[ERROR] testTimeLengthQuery2(org.ballerinalang.stdlib.streams.BallerinaStreamsV2TimeLengthWindowTest)  Time elapsed: 0.028 s  <<< FAILURE!
org.ballerinalang.util.exceptions.BLangRuntimeException: 
error: {ballerina/task}TaskError {message:"Cannot initialize the Task Listener/Scheduler."}
	at ballerina/task:init(<native>)
	   ballerina/task:__init(listener.bal:30)
	   ballerina/task:__init(scheduler.bal:22)
	   ballerina/streams:__init(scheduler.bal:35)
	   ballerina/streams:__init(windows.bal:1448)
	   ballerina/streams:timeLength(windows.bal:1613)
	   testTimeLengthwindow(streamingv2-time-length-window-test2.bal:78)
	   startTimeLengthwindowTest2(streamingv2-time-length-window-test2.bal:55)
	at org.ballerinalang.stdlib.streams.BallerinaStreamsV2TimeLengthWindowTest.testTimeLengthQuery2(BallerinaStreamsV2TimeLengthWindowTest.java:66)

[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Failures: 
[ERROR]   BallerinaStreamsV2StatePersistenceTest.testStreamingStatePersistence:89 expected [60] but found [30]
[ERROR]   BallerinaStreamsV2TimeLengthWindowTest.testTimeLengthQuery2:66 Â» BLangRuntime ...
[INFO] 

```

**Comment by Manuri** [comment]
[streaming-test-failure-log.txt](https://github.com/ballerina-platform/ballerina-lang/files/3023779/streaming-test-failure-log.txt)

**Comment by gimantha** [comment]
Fixed with #14848
